README.md

```diff
@@ -62,6 +62,7 @@ This class allows specifying the following parameter:
    * `manage_users_deny` - optional - defaults to false, whether to manage `/etc/cron.deny`
    * `users_allow` - optional - An array of users to add to `/etc/cron.allow`
    * `users_deny` - optional - An array of users to add to `/etc/cron.deny`
+   * `merge` - optional - default to "hash"
```

Comment:
I'm not sure if the term `optional` is correct here? The parameter is an enum and defaults to the string `hash` and not undef.

Reply:
Yeah, I just removed that whole redundant list.